alica_engine/src/engine/model/VariableBinding.cpp

```diff
@@ -15,7 +15,25 @@ VariableBinding::VariableBinding()
 {
 }
 
-VariableBinding::~VariableBinding() {}
+void VariableBinding::setSubPlan(AbstractPlan* subPlan)
+{
+    if (_subPlan)
```

Comment:
{} after any conditional.  Otherwise it is easy to get a bug by adding a new line

Reply:
My concern is more about, which level of warnings should be here: ignore, warning message, die.